cfme/fixtures/intelligence.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+import pytest
+
+from cfme.utils.conf import cfme_data
+from cfme.utils.ftp import FTPClientWrapper
+from cfme.utils.ftp import FTPException
+from cfme.utils.log import logger
+
+
+@pytest.fixture
+def yaml_path(request, yaml_name):
```

Comment:
It looks like this was committed/pushed maybe halfway through writing?

This new fixture isn't registered in `cfme/test_framework/pytest_plugin.py`, which means its not available to any test modules.

The function still exists in test_import_export_reports_widgets.py below, not decorated as a fixture.

Fixtures should not take arguments, as they aren't called directly by the test (unless they return a function object).

Are you planning on writing a generic fixture that can be used by multiple test modules? If so they'll have to parametrize on yaml_name, not try to pass it or call this method directly.

Reply:
Oh yes, I committed this one by mistake. I was working on writing a global fixture for reports and schedules. Removing it :grimacing: